test/test_helper.exs

```diff
@@ -0,0 +1 @@
+Application.ensure_all_started(:exla)
```

Comment:
Why do we need to start explicitly?

Reply:
Earlier Elixir versions did not start optional applications.